engine/admin/index.md

```diff
@@ -115,23 +115,18 @@ as the process manager. As of Ubuntu 15.04, Ubuntu uses `systemd` as its process
 manager. For Ubuntu 15.04 and higher, refer to [control and configure Docker
 with systemd](systemd.md).
 
-You configure the `docker` daemon in the `/etc/default/docker` file on your
-system. You do this by specifying values in a `DOCKER_OPTS` variable.
-
 To configure Docker options:
 
 1. Log into your host as a user with `sudo` or `root` privileges.
 
-2. If you don't have one, create the `/etc/default/docker` file on your host. Depending on how
-you installed Docker, you may already have this file.
-
-3.  Open the file with your favorite editor.
+2. Create or edit the `/etc/default/docker` file on your host. If
+   you use `systemd`, create or edit `/etc/systemd/docker` instead.
```

Comment:
This is not correct; we don't have a configuration file at this location for `systemd` based distros. For systemd based distros;

1. use a daemon.json configuration file https://docs.docker.com/engine/reference/commandline/dockerd/#/daemon-configuration-file
2. or use systemd drop-in files; https://docs.docker.com/engine/admin/systemd/#/custom-docker-daemon-options


Option 1. is most flexible as it allows reloading some configuration changes without restarting the daemon, and does not require knowledge about systemd (e.g. not required to `sudo systemctl daemon-reload` before `sudo systemctl restart docker`). Also, the `daemon.json` can be used in all cases, independent of the init system in use.